app/javascript/components/concept-map/Concept.tsx

```diff
@@ -50,6 +57,9 @@ export const Concept = ({
   if (isActive) {
     classes.push('active')
   }
+  if (isActiveHover) {
+    classes.push('hover')
```

Comment:
What does this class do?

Reply:
Nothing anymore, my plan was to remove it